components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "defectdojo",
-  "version": "2.9.0-dev",
+  "version": "2.9.0",
```

Comment:
This should not be in the PR

Reply:
I will close this PR and create a new one, somehow VS Code rebased the branch with accept both.